client/js/app/components/explorer/visualization/keen_viz.js

```diff
@@ -22,9 +22,12 @@ var KeenViz = React.createClass({
       .el(this.refs['keen-viz'])
       .height(300)
       .title(null)
-      .sortGroups('desc')
       .type(this.props.model.metadata.visualization.chart_type);
 
+    if (!this.props.model.query.analysis_type === "funnel") {
```

Comment:
This should be:
```
if (this.props.model.query.analysis_type !== "funnel") {
```

In the current form, this line evaluates the truthyness of `this.props.model.query.analysis_type` and *then* compares that result (`true` or `false`) for strict equality against `"funnel"`. But what you want to do here is check for strict equality between two strings.